keras_cv/models/segmentation/deeplab_v3/deeplab_v3.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_cv.layers.spatial_pyramid import SpatialPyramidPooling
+from keras_cv.models.legacy import utils
+from keras_cv.models.task import Task
+
+
+@keras.utils.register_keras_serializable(package="keras_cv")
+class DeepLabV3(Task):
+    """A Keras model implementing the DeepLabV3 architecture for semantic
+    segmentation.
+
+    References:
+        - [Rethinking Atrous Convolution for Semantic Image Segmentation](https://arxiv.org/abs/1706.05587)
+        (CVPR 2017)
+
+    Examples:
```

Comment:
Move `Examples:` section to end of the docstring

Reply:
Should it be moved after the `Args:` section?

Reply:
Yes, thank you!